cmd/root.go

```diff
@@ -380,7 +379,7 @@ func handleHelm(kubeContext string) (*output.Output, error) {
 		}
 		packages, err := ahClient.List()
 		if err != nil {
-			return nil, fmt.Errorf("Error getting artifacthub package repos: %v", err)
+			return nil, fmt.Errorf("error getting artifacthub package repos: %v", err)
```

Comment:
Isn't the use of a capital E more correct in this situation?  Start of a sentence.

Reply:
This is a common linting suggestion for golang, that log messages shouldn't be full sentences and should be lowercase.